src/models/model.cpp

```diff
@@ -415,29 +423,6 @@ void model::add_metric(metric *m) {
   m_metrics.push_back(m);
 }
 
-void model::replace_weights(std::vector<weights*>& new_weights) {
```

Comment:
I am trying to remember if there is any reason that this would still be necessary.  I believe that it was used in the previous multi-model approach, where weights were moved.